package.json

```diff
@@ -13,7 +13,7 @@
     "export": "rm -rf dist/ && next export -o dist/"
   },
   "dependencies": {
-    "@livepeer/react": "^1.1.0",
+    "@livepeer/react": "^1.5.1",
```

Comment:
Oh it was pretty outdated. We should move this to livepeer.js repo to keep it close together. I can help with the fly.io CI if/when we ever do that.

Reply:
Sounds good, I'll do that soon!